src/app.js

```diff
@@ -142,9 +142,11 @@ router.post('/action', verifyRequest, actionHandler(async (action) => {
 		case 'block_actions': {
 			const [answer, date] = action.actions[0].value.split('_');
 			const channel = action.channel.id;
+			const slackId = action.container.message_ts;
 			const timestamp = Date.now();
 			logger.debug({user, channel, date, answer, timestamp}, 'Vote action');
-			await dynamo.vote({channel, date, user, answer, timestamp});
+			await dynamo.setSlackId({channel, date, slackId});
```

Comment:
Need to set the id here or we get duplicated messages in slack.

When setting the slackId on creation it is way too slow. See test channel.

Might be a better way to do it.